scripts/uberenv/spack_configs/blueos_3_ppc64le_ib_p9/devtools/packages.yaml

```diff
@@ -22,86 +21,99 @@ packages:
       lapack: [netlib-lapack]
       mpi: [mvapich2]
   cuda:
-    buildable: False
+    buildable: false
   mvapich2:
-    buildable: False
+    buildable: false
   openmpi:
-    buildable: False
+    buildable: false
   mpich:
-    buildable: False
+    buildable: false
   spectrum-mpi:
-    buildable: False
+    buildable: false
   charmpp:
-    buildable: False
+    buildable: false
   charm:
-    buildable: False
+    buildable: false
   intel-mpi:
-    buildable: False
+    buildable: false
   intel-parallel-studio:
-    buildable: False
+    buildable: false
   fujitsu-mpi:
-    buildable: False
+    buildable: false
   mpilander:
-    buildable: False
+    buildable: false
   mpt:
-    buildable: False
+    buildable: false
```

Comment:
```
mpi:
  buildable: false
```
?

Reply:
Apparently mpt is another mpi implementation.  Are these even needed at this point? half my spack configs dont list them.  It could just be a relic of a bygone era

Reply:
Sorry, I meant Spack now support "buildable: false" at the provider level.
So you could replace all this by the snippet above.

Reply:
It’s still useful to prevent building any mpi provider, since that’s the only way to force spack to use the one specified as external or fail, _in any circumstances_.

Reply:
woo! cleaning up spack configs even more! they are almost readable at this point.